src/guidescan_web/genomics/resolver.clj

```diff
@@ -177,9 +177,9 @@
     (if-let [conns (get-in config [:config :sequence-resolvers])]
       (do
         (timbre/info "Checking sequence-resolvers")
-        (for [organism (keys conns)]
+        (doseq [organism (keys conns)]
```

Comment:
I noticed that the checking-sequence-servers logic was not firing. My clojure is very basic, but something like `for` never actually runs in this case while `doseq` causes eager evaluation so it does.